tutorials/fit/fitNormSum.py

```diff
@@ -0,0 +1,100 @@
+## \file
+## \ingroup tutorial_fit
+## \notebook
+## Tutorial for normalized sum of two functions
+## Here: a background exponential and a crystalball function
+## Parameters can be set:
+##  1.   with the TF1 object before adding the function (for 3) and 4))
+##  2.  with the TF1NormSum object (first two are the coefficients, then the non constant parameters)
+##  3. with the TF1 object after adding the function
+##
+## Sum can be constructed by:
+##  1. by a string containing the names of the functions and/or the coefficient in front
+##  2. by a string containg formulas like expo, gaus...
+##  3. by the list of functions and coefficients (which are 1 by default)
+##  4. by a std::vector for functions and coefficients
+##
+## \macro_image
+## \macro_output
+## \macro_code
+##
+## \author Jonas Rembser, Lorenzo Moneta (C++ version)
+
+import ROOT
+
+nsig = 50000
+nbkg = 1000000
+nEvents = nsig + nbkg
+nBins = 1000
+
+signal_mean = 3.0
+f_cb = ROOT.TF1("MyCrystalBall", "crystalball", -5.0, 5.0)
+f_exp = ROOT.TF1("MyExponential", "expo", -5.0, 5.0)
+
+# I.:
+f_exp.SetParameters(1.0, -0.3)
+f_cb.SetParameters(1, signal_mean, 0.3, 2, 1.5)
+
+# CONSTRUCTION OF THE TF1NORMSUM OBJECT ........................................
+# 1) :
+fnorm_exp_cb = ROOT.TF1NormSum(f_cb, f_exp, nsig, nbkg)
+# 4) :
+
+f_sum = ROOT.TF1("fsum", fnorm_exp_cb, -5.0, 5.0, fnorm_exp_cb.GetNpar())
+
+# III.:
+parameter_values = fnorm_exp_cb.GetParameters()
+f_sum.SetParameters(parameter_values.data())
+# Note: in the C++ tutorial, the parameter value sync is done in one line with:
+# ```C++
+#     f_sum->SetParameters(fnorm_exp_cb->GetParameters().data());
+# ```
+# However, GetParameters() returns in std::vector by value, which doesn't
```

Comment:
Thank you for commenting on this.
In C++ GetParameters returns a pointer to an array (double *) stored in TF1. In Python it returns a copy as an std::vector that will be deleted?

Reply:
No, in case of the `TF1NormSum` it always returns a vector by value also in C++: hist/hist/inc/TF1NormSum.h

Reply:
I have now mentioned in the inline comment that this is specifically `TF1NormSum::GetParameters()`.